src/java/us/kbase/mobu/tester/test/CallbackServerTest.java

```diff
@@ -405,7 +405,8 @@ public void maxJobs() throws Exception {
             res.callMethod("njs_sdk_test_1.run", params, "dev");
         } catch (ServerException se) {
             assertThat("incorrect error message", se.getLocalizedMessage(),
-                    is("No more than 10 concurrently running methods are allowed"));
+                    // you've got to be shitting me right now with this quoting shit wtf
+                    is("\"'No more than 10 concurrently running methods are allowed'\""));
```

Comment:
So the CBS makes the string
"'No more than 10....'"?
With single and double quotes inline?
That's something.